### PR TITLE
openthread: samples: Reorganise openthread cert overlay

### PR DESCRIPTION
**Pull request by canisLupus1313**
This commit removes BLE functionality from overlay-cert.conf
and delagates this to separate overaly.

Signed-off-by: Przemyslaw Bida <przemyslaw.bida@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/ug_thread_certification.rst [hunk 1]
```diff
@@ -64,24 +64,27 @@ Complete the following steps to prepare for the certification tests:
 
 #. Build the certification image.
 
-   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` overlay file.
+   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` plus :file:`overlay-multiprotocol.conf` overlay files.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:67]:
```suggestion
   The :ref:`ot_cli_sample` sample is used as a base, modified with the :file:`harness/overlay-cert.conf` and :file:`overlay-multiprotocol.conf` overlay files.
```

**Review by edmont — APPROVED** [review]

File: doc/nrf/ug_thread_ot_memory.rst [hunk 2]
```diff
@@ -20,7 +20,7 @@ The memory requirement tables list flash memory and RAM requirements for the :re
 The values change depending on device type and hardware platform.
 Moreover, take into account the following considerations:
 
-* The sample was compiled using the default :file:`prj.conf` plus :file:`harness\overlay-cert.conf` plus the corresponding :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY` and :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
+* The sample was compiled using the default :file:`prj.conf` plus :file:`harness\overlay-cert.conf` plus the corresponding plus :file:`overlay-multiprotocol.conf` :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY` and :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_ot_memory.rst:23]:
```suggestion
* The sample was compiled using the default :file:`prj.conf` plus :file:`harness\overlay-cert.conf` and :file:`overlay-multiprotocol.conf` overlay files, and the corresponding :kconfig:option:`OPENTHREAD_NORDIC_LIBRARY` and :kconfig:option:`OPENTHREAD_THREAD_VERSION` choices.
```

File: doc/nrf/ug_thread_certification.rst [hunk 3]
```diff
@@ -64,24 +64,27 @@ Complete the following steps to prepare for the certification tests:
...
 
-     If the overlay file is not visible in the drop-down menu, navigate to :file:`ncs/nrf/samples/openthread/cli/harness/` and copy the :file:`overlay-cert.conf` file to :file:`ncs/nrf/samples/openthread/cli/` and try again.
+     If the :file:`harness/overlay-cert.conf` is not visible in the drop-down menu, navigate to :file:`ncs/nrf/samples/openthread/cli/harness/` and copy the :file:`overlay-cert.conf` file to :file:`ncs/nrf/samples/openthread/cli/` and try again.
 
    .. note::
       The configuration option selects the precompiled OpenThread libraries.
       The overlay file enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth® LE advertising.
 
+   .. note::
+      The overlay :file:`overlay-multiprotocol.conf` is not supported with nrf52833dk_nrf52833.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:86]:
```suggestion
      The overlay :file:`overlay-multiprotocol.conf` is not supported with ``nrf52833dk_nrf52833``.
```

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
A couple of small changes needed.

File: doc/nrf/ug_thread_certification.rst [hunk 4]
```diff
@@ -64,24 +64,27 @@ Complete the following steps to prepare for the certification tests:
...
 
    * If building using Visual Studio Code, you must first `create the application <Creating an application_>`_ using the CLI sample, and then `build the application <Building an application_>`_.
-     Select the :file:`harness/overlay-cert.conf` overlay file in the :guilabel:`Kconfig fragment` drop-down menu and add ``CONFIG_OPENTHREAD_LIBRARY=y`` to the :guilabel:`Additional CMake arguments` text field.
+     Select the :file:`harness/overlay-cert.conf` and :file:`overlay-multiprotocol.conf` overlay files in the :guilabel:`Kconfig fragment` drop-down menu and add ``CONFIG_OPENTHREAD_LIBRARY=y`` to the :guilabel:`Additional CMake arguments` text field.
 
-     If the overlay file is not visible in the drop-down menu, navigate to :file:`ncs/nrf/samples/openthread/cli/harness/` and copy the :file:`overlay-cert.conf` file to :file:`ncs/nrf/samples/openthread/cli/` and try again.
+     If the :file:`harness/overlay-cert.conf` is not visible in the drop-down menu, navigate to :file:`ncs/nrf/samples/openthread/cli/harness/` and copy the :file:`overlay-cert.conf` file to :file:`ncs/nrf/samples/openthread/cli/` and try again.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:79]:
```suggestion
     If the :file:`harness/overlay-cert.conf` overlay file is not visible in the drop-down menu, navigate to :file:`ncs/nrf/samples/openthread/cli/harness/` and copy the :file:`overlay-cert.conf` file to :file:`ncs/nrf/samples/openthread/cli/` and try again.
```

File: doc/nrf/ug_thread_certification.rst [hunk 5]
```diff
@@ -64,24 +64,27 @@ Complete the following steps to prepare for the certification tests:
...
 
    .. note::
       The configuration option selects the precompiled OpenThread libraries.
       The overlay file enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth® LE advertising.
 
+   .. note::
+      The overlay :file:`overlay-multiprotocol.conf` is not supported with ``nrf52833dk_nrf52833``.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:86]:
```suggestion
      The :file:`overlay-multiprotocol.conf` overlay file is not supported with ``nrf52833dk_nrf52833``.
```

File: doc/nrf/ug_thread_certification.rst [hunk 6]
```diff
@@ -64,24 +64,27 @@ Complete the following steps to prepare for the certification tests:
...
-     Select the :file:`harness/overlay-cert.conf` overlay file in the :guilabel:`Kconfig fragment` drop-down menu and add ``CONFIG_OPENTHREAD_LIBRARY=y`` to the :guilabel:`Additional CMake arguments` text field.
+     Select the :file:`harness/overlay-cert.conf` and :file:`overlay-multiprotocol.conf` overlay files in the :guilabel:`Kconfig fragment` drop-down menu and add ``CONFIG_OPENTHREAD_LIBRARY=y`` to the :guilabel:`Additional CMake arguments` text field.
 
-     If the overlay file is not visible in the drop-down menu, navigate to :file:`ncs/nrf/samples/openthread/cli/harness/` and copy the :file:`overlay-cert.conf` file to :file:`ncs/nrf/samples/openthread/cli/` and try again.
+     If the :file:`harness/overlay-cert.conf` is not visible in the drop-down menu, navigate to :file:`ncs/nrf/samples/openthread/cli/harness/` and copy the :file:`overlay-cert.conf` file to :file:`ncs/nrf/samples/openthread/cli/` and try again.
 
    .. note::
       The configuration option selects the precompiled OpenThread libraries.
       The overlay file enables :ref:`multiprotocol support <ug_multiprotocol_support>` with Bluetooth® LE advertising.
```
Comment (wiba-nordic) [inline doc/nrf/ug_thread_certification.rst:83]: Which overlay file? Or both?

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
I missed a couple places last time.

**Review by wiba-nordic — APPROVED** [review]

**Review by lmaciejonczyk — APPROVED** [review]

**Comment by lmaciejonczyk** [comment]
Changelog-entry is still missing.

**Review by rlubos — APPROVED** [review]

**Comment by wiba-nordic** [comment]
Changelog entry looks good.

**Comment by edmont** [comment]
Please rebase.

**Pull request merged**